src/main/java/core/basesyntax/BinaryTree.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+class BinaryTree {
+    // Root of Binary Tree
+    private Node root;
+
+    public BinaryTree() {
+        root = null;
+    }
+
+    public Node getRoot() {
+        return root;
+    }
+
+    public void setRoot(Node root) {
+        this.root = root;
+    }
+
+    /* Given a binary tree, print its nodes according to the 
+      "bottom-up" postorder traversal. */
+    public void printPostOrder(Node node) {
```

Comment:
Let's return List of Nodes (and rename obviously). Then you'll be able to unit test this method.